lib/domain/usecase/create_room_interactor.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:dartz/dartz.dart';
+import 'package:fluffychat/di/global/get_it_initializer.dart';
+import 'package:fluffychat/domain/app_state/room/create_room_failed.dart';
+import 'package:fluffychat/domain/app_state/room/create_room_success.dart';
+import 'package:fluffychat/domain/app_state/room/set_room_avatar_failed.dart';
+import 'package:fluffychat/domain/app_state/room/set_room_avatar_success.dart';
+import 'package:fluffychat/domain/model/contact/new_room_informations.dart';
+import 'package:fluffychat/domain/usecase/set_room_avatar_interactor.dart';
+import 'package:matrix/matrix.dart';
+
+class CreateRoomInteractor {
+  final setRoomAvatarInteractor = getIt.get<SetRoomAvatarInteractor>();
+
+  Stream<Either<CreateRoomFailed, CreateRoomSuccess>> execute(

```

Comment:
Try to make it general, as I said, to create a group we need to integrate some other actors (inter-actors). Some belongs to `CreateRoom` but some other does not (Fx: Change avatar, ....)
```suggestion
  Stream<Either<Failure, Success>> execute(

```